prediction_market_agent/tools/anvil/anvil_requests.py

```diff
@@ -0,0 +1,45 @@
+import requests
+from prediction_market_agent_tooling.gtypes import HexBytes, xdai_type
+from prediction_market_agent_tooling.loggers import logger
+from prediction_market_agent_tooling.tools.web3_utils import xdai_to_wei
+from web3 import Web3
+
+
+def get_balance(rpc_url: str, address: str, block: int) -> tuple[int, int]:
```

Comment:
We already have [get_balances](https://github.com/gnosis/prediction-market-agent-tooling/blob/b81a3f7fe7b84dfc6a380bdcd0303964280ab395/prediction_market_agent_tooling/tools/balances.py#L26) in PMAT, can't blockNumber be used there instead of having duplicated function?

Reply:
True - but not used, hence deleted